### PR TITLE
Scores table - student name/id spacing, truncation, tooltip

### PR DESCRIPTION
**Pull request by darrenmason**
tooltip

non-CC:

image

### CODE REVIEW

**Comment by philschatz** [comment]
Can the non-CC scores not contain the extra student id column since High school teachers do not know about it?

File: resources/styles/components/scores/index.less [hunk 1]
```diff
@@ -162,17 +162,20 @@
       display: inline-block;
       width: 50%;
       text-align: center;
+      overflow: hidden;
+    text-overflow: ellipsis;
```
Comment (philschatz) [inline resources/styles/components/scores/index.less]: looks like these are not indented properly

File: resources/styles/components/scores/index.less [hunk 2]
```diff
@@ -162,17 +162,20 @@
...
+      
       &:first-child {
         border-right: 1px solid @nav-tabs-border-color;
       }
     }
   }
-  .name-cell .name {
-    padding-left: 1rem;
-    text-align: left;
-    display: inline-block;
```
Comment (nathanstitt) [inline resources/styles/components/scores/index.less]: The `text-align: left;` rule here was used to remove the default center alignment for the name column.  I think we probably want to retain that :grin:

**Comment by philschatz** [comment]
Thanks for the updates!

**Pull request merged**